spec/php-spec.coffee

```diff
@@ -1863,6 +1863,28 @@ describe 'PHP grammar', ->
     expect(tokens[4]).toEqual value: '\'', scopes: ['source.php', 'string.quoted.single.php', 'punctuation.definition.string.end.php']
     expect(tokens[5]).toEqual value: ';', scopes: ['source.php', 'punctuation.terminator.expression.php']
 
+  it 'should tokenize heredoc in a function call correctly', ->
+    lines = grammar.tokenizeLines  '''
+    foo(
+      <<<HEREDOC
+      This is just a cool test.
+      HEREDOC,
+      $bar
+    );
+    '''
+
+    expect(lines[0][0]).toEqual value: 'foo', scopes: ['source.php', 'meta.function-call.php', 'entity.name.function.php']
+    expect(lines[0][1]).toEqual value: '(', scopes: ['source.php', 'meta.function-call.php', 'punctuation.definition.arguments.begin.bracket.round.php']
+    expect(lines[1][1]).toEqual value: '<<<', scopes: ['source.php', 'meta.function-call.php', 'string.unquoted.heredoc.php', 'punctuation.definition.string.php']
```

Comment:
lines[1][0] is missing

Reply:
> lines[1][0] is missing

I just removed those intentions because I thought it was unnecessary as earlier examples omitted indention as well. if u think it is better to add them so I will ...

Reply:
That's fine then, my apologies. The offset should be enough to indicate if something's wrong in the future.

Reply:
And btw,
Since I'm new to whole GitHub, Do I need to force push and make a cleaner commit or just leave things as they are right now ?

Reply:
You can do a force push if you want to. But users who have the ability to merge pull requests, can choose to squash noisy pull requests themselves. See https://help.github.com/en/articles/merging-a-pull-request#merging-a-pull-request-on-github

Reply:
I guess, I learned enough for now ...
I've been a programmer since 14 and I was afraid of git till recent year.
Thank you @Ingramz